.github/workflows/self-hosted-early-features-pr-bot.yml

```diff
@@ -0,0 +1,54 @@
+name: self-hosted early features PR bot
+
+on:
+  pull_request:
+    types: [opened, edited]
+
+jobs:
+  sync:
+    name: make comment on missing EA'd features in self-hosted
+    runs-on: ubuntu-latest
+    permissions:
+      issues: write
+      pull-requests: write
+    steps:
+      - uses: actions/checkout@93ea575cb5d8a053eaa0ac8fa3b40d7e05a33cc8 # v3.1.0
+
+      - name: getsentry token
+        uses: getsentry/action-github-app-token@97c9e23528286821f97fba885c1b1123284b29cc # v2.0.0
+        id: getsentry
+        with:
+          app_id: ${{ vars.SENTRY_INTERNAL_APP_ID }}
+          private_key: ${{ secrets.SENTRY_INTERNAL_APP_PRIVATE_KEY }}
+
+      - name: Make API Request
+        id: api_request
+        run: |
+          response=$(curl -s -H "Authorization: Bearer ${{ steps.getsentry.outputs.token }}" 'https://https://sentry.sentry.io/api/0/internal-stats/ea-feature-flags/')
```

Comment:
I don't think this is the token you're looking for.  this is a token for **github** and you're making a request to **sentry**